design-documents/graph-ql/coverage/ProductPrices.graphqls

```diff
@@ -0,0 +1,52 @@
+interface ProductInterface{
+    ...
+    price: ProductPrices @deprecated (use price_range)
+    price_range: PriceRange!
+    tier_price: Float @deprecated (use tier_prices)
+    tier_prices: [ProductTierPrices]
+    ...
+}
+
+type ProductPrices { @deprecated (use PriceRange)
+    minimalPrice: @deprecated
+    maximalPrice: @deprecated
+    regularPrice: @deprecated
+}
+
+type ProductTierPrices {
+    customer_group_id: String
+    qty: Float
+    value: Float @deprecated (use final_price)
+    final_price: Money
+    percentage_value: Float @deprecated (use discount)
+    discount: ProductDiscount
+    website_id: Float
```

Comment:
Do we need to have `website_id` as part of prices?
Generally, the website should be resolved by Store header